Applications/IntelliDivide/Contracts/Common/OptimizationBasePart.cs

```diff
@@ -67,19 +67,19 @@ public OptimizationBasePart(UnitSystem unitSystem) : this()
         #region (2) Cutting / Nesting
 
         /// <inheritdoc />
-        [ValueDependsOnUnitSystem(BaseUnit.Millimeter)]
+        [ValueDependsOnUnitSystem(BaseUnit.Millimeter, PropertyDecimals.EdgebandingThicknessMillimeterDecimals, PropertyDecimals.InchDecimals)]
```

Comment:
Does this need to get specified in all cases? For Millimeter ist would be in most cases equal.

Reply:
As Robin is on holiday I updated it. Please check again